index.bs

```diff
@@ -1121,6 +1143,9 @@ When this operation is invoked, the authenticator must perform the following pro
 - Prompt the user for consent to create a new credential. The prompt for obtaining this consent is shown by the authenticator
     if it has its own output capability, or by the user agent otherwise. If the user denies consent, return an error code
     equivalent to NotAllowedError and terminate the operation.
+- If the {{ScopedCredentialOptions/verificationGesture}} parameter is set to true, prompt the user for performing an 
+    <a>authorization gesture</a> to verify the user. If this prompt is shown with sufficient explanation that the success of 
```

Comment:
["Authorization gesture"](https://w3c.github.io/webauthn/#authorization-gesture) is defined as "Essentially the same as user verification." Please only use one term for a given concept.